__init__.py

```diff
@@ -327,6 +327,18 @@ def _init_setup_options(self):
                 {"name": "Dutch", "code": "nl", "system_code": "nl_NL"}
             ]
 
+        if "backend_list" not in self.settings:
+            # backend_name: display name to display in UI
+            # backend_icon: display icon to display in UI
+            # backend_type: backend type for configuration
+            self.settings["backend_list"] = [
+                {"backend_name": "No Backend", "backend_icon": "icons/nobackend.svg", "backend_type": "offline"},
+                {"backend_name": "Personal Backend", "backend_icon": "icons/personal.svg", "backend_type": "personal"},
+                {"backend_name": "OpenVoice Backend", "backend_icon": "icons/ovos.svg", "backend_type": "ovos"},
```

Comment:
For consistency i think this should say OpenVoiceOS Backend, we are using OVOS or OpenVoiceOS everywhere. just Open Voice can be confusing since theres many projects using that wording for various different things